pkg/virt-controller/services/renderresources.go

```diff
@@ -646,6 +646,43 @@ func hotplugContainerRequests(config *virtconfig.ClusterConfig) k8sv1.ResourceLi
 	}
 }
 
+func vmExportContainerResourceRequirements(config *virtconfig.ClusterConfig) k8sv1.ResourceRequirements {
+	return k8sv1.ResourceRequirements{
+		Limits:   vmExportContainerLimits(config),
+		Requests: vmExportContainerRequests(config),
+	}
+}
+
+func vmExportContainerLimits(config *virtconfig.ClusterConfig) k8sv1.ResourceList {
+	cpuQuantity := resource.MustParse("1")
+	if cpu := config.GetSupportContainerLimit(v1.VMExport, k8sv1.ResourceCPU); cpu != nil {
+		cpuQuantity = *cpu
+	}
+	memQuantity := resource.MustParse("1024M")
+	if mem := config.GetSupportContainerLimit(v1.VMExport, k8sv1.ResourceMemory); mem != nil {
+		memQuantity = *mem
+	}
+	return k8sv1.ResourceList{
+		k8sv1.ResourceCPU:    cpuQuantity,
+		k8sv1.ResourceMemory: memQuantity,
+	}
```

Comment:
So we're now always setting memory/CPU limits, even when not specified in the CR, right?
Is there a chance that these values will be too small? If so, isn't that a problem? If not, why make them configurable?

Reply:
Yes we are always setting them, the export server itself doesn't use a lot of memory, it is just about caches and things while compressing the image that is being exported. This is why the request is relatively low compared to the limit. CPU: 100m vs CPU: 1 and memory: 200Mi vs memory: 1024Mi. The reason for allowing configuration is that some people want a very specific request to limit ratio, and this allows them that option.